roles/container-engine/containerd/templates/containerd.service.j2

```diff
@@ -31,6 +31,7 @@ RestartSec=5
 LimitNPROC=infinity
 LimitCORE=infinity
 LimitNOFILE=infinity
+LimitMEMLOCK=infinity
```

Comment:
When reading the corresponding issue https://github.com/kubernetes-sigs/kubespray/issues/9267 I guessed we need to add an option to customize this value and the default value should be `infinity`.

I am ok to add the static value like this as the other values, but I just want to make sure before merging.

Reply:
Yes actually it would be cool to be able to configure these values through kubespray variables. I also found an issue on CentOS 9 which maybe requires to modify the NOFILE limit (see https://stackoverflow.com/questions/73185002/yum-update-stucks-inside-docker).

Reply:
@oomichi @sli720 
😊 I have updated the code, please help to review it again.